### ISSUE TITLE
Cannot play anything after fresh setup

### ISSUE THREAD

**Issue by andbuitra** [jellyfin/jellyfin-kodi#952]
I have reinstalled Kodi and the Jellyfin for Kodi plugin and everything goes smoothly regarding the setup process but when I attempt to play content it fails  saying "Jellyfin error. Check log for more details" and "Playback failed. One or more items failed to play...".

Checking the it says:

```
2024-12-15 20:13:36.160 T:746     error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'IndexError'>
                                                   Error Contents: list index out of range
                                                   Traceback (most recent call last):
                                                     File "/home/andres/.var/app/tv.kodi.Kodi/data/addons/plugin.video.jellyfin/default.py", line 18, in <module>
                                                       LOG.debug("--->[ default ]")
                                                       ^^^^^^^^^
                                                     File "/home/andres/.var/app/tv.kodi.Kodi/data/addons/plugin.video.jellyfin/jellyfin_kodi/helper/lazylogger.py", line 18, in __getattr__
                                                       from .loghandler import getLogger
                                                     File "/home/andres/.var/app/tv.kodi.Kodi/data/addons/plugin.video.jellyfin/jellyfin_kodi/helper/loghandler.py", line 127, in <module>
                                                       __LOGGER.addHandler(LogHandler())
                                                                           ^^^^^^^^^^^^
                                                     File "/home/andres/.var/app/tv.kodi.Kodi/data/addons/plugin.video.jellyfin/jellyfin_kodi/helper/loghandler.py", line 47, in __init__
                                                       self.sensitive["Server"].append(server["address"].split("://")[1])
                                                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
                                                   IndexError: list index out of range
                                                   -->End of Python script error report<--
```

and

```
2024-12-15 20:13:36.188 T:7       error <general>: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.jellyfin/f137a2dd21bbc1b99aa5c0f6bf02a805/?filename=[redacted]]
2024-12-15 20:13:36.188 T:7       debug <general>: Playlist Player: no more playable items... aborting playback
```

I have setup the plugin in the past and it was working until recently where I had to factory reset my chromecasts and now it won't connect. This particular log comes from the desktop client (flatpak) but it's the same error on all devices.

Jellyfin Server version: 10.10.3 (but it was also tested with 10.10.1) running on Docker with the LSIO image.

When configuring the URL I have explicitly typed the "https://" and the hostname that I use for Jellyfin. I have also tried with the hostname and IP for the server where Jellyfin resides and the exposed port with no luck. I have setup the add-on plenty of times and the configuration I've done (following Jellyfin's guide) has worked until today. 

--

In summary, configuring the plugin is painless and the content is correctly being listed but playback is failing. I've tried removing the library and re-adding it with no luck as well. There's no relevant messages on the server side either. It just confirms that authentication was successful but doesn't say anything else when trying to play content.

**Comment by ulrichsc** [comment]
I have the same issue after a fresh installation on a new Pi5 with libreelec. Playback only works if I enable forced transcoding.

**Comment by andbuitra** [comment]
Something may have changed from server side or add-on side but I just tried it again a couple of weeks ago and it worked okay when setting it up fresh in my Chromecasts.

I had a Kodi client setup in my PC from when this issue started happening and today I opened it to find the same error. The server has updated from then but with current server version and that config, it kept failing. I just uninstalled it, install it again and it started working ok.

I'll close this issue but I don't really understand what the source problem was

**Comment by oddstr13** [comment]
99% sure the error message was caused by unexpected data in a server setting

**Comment by oddstr13** [comment]
duplicate of #734
